akka-sample-sharding-scala/killrweather/src/main/scala/sample/killrweather/WeatherHttpServer.scala

```diff
@@ -0,0 +1,52 @@
+package sample.killrweather
+
+import scala.util.{Failure, Success}
+import scala.concurrent.duration._
+
+import akka.actor.typed.ActorSystem
+import akka.actor.CoordinatedShutdown
+import akka.{Done, actor => classic}
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.server.Route
+
+private[killrweather] object WeatherHttpServer {
```

Comment:
added a Http here to make it clear what kind of server it is even without opening the file